### ISSUE TITLE
Dead link in README.md

### ISSUE THREAD

**Issue by gihanmarasingha** [kovzol/geogebra-discovery#6]
The link to https://www.geogebra.org/team in the README.md file is dead.

**Comment by kovzol** [comment]
Thanks for reporting, we are on it. Hopefully there is an official link that points to something like https://web.archive.org/web/20200313073454/https://www.geogebra.org/team.

**Comment by kovzol** [comment]
No, there is no official link. I temporarily changed the link in a7cd018.
